sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/encoders/ExpressionEncoderSuite.scala

```diff
@@ -652,7 +652,7 @@ class ExpressionEncoderSuite extends CodegenInterpretedPlanTest with AnalysisTes
             val e = intercept[RuntimeException] {
               toRow(bigNumeric)
             }
-            assert(e.getMessage.contains("Error while encoding"))
+            assert(e.getMessage.contains("Failed to encode a value of the expressions:"))
```

Comment:
Maybe this is a chance to use `checkError` with `errorClass`. cc @MaxGekk

Reply:
Yep, if it is possible let's use `checkError` but this one is from a nested non-Spark exception, isn't.